src/content/docs/release-notes/agent-release-notes/python-release-notes/python-agent-100300.mdx

```diff
@@ -0,0 +1,64 @@
+
+The Python agent now allows you to opt-in to adding your custom tags (labels) to agent-forwarded logs. With custom tags on logs, platform engineers can easily filter, search, and correlate log data for faster and more efficient troubleshooting, improved performance, and optimized resource utilization. To learn more about this feature see the [documentation](/docs/logs/logs-context/Custom-tags-agent-forwarder-logs).
+ 
+Install the agent using `easy_install/pip/distribute` via the [Python Package Index](https://pypi.python.org/pypi/newrelic) or download it directly from the [New Relic download site](https://download.newrelic.com/python_agent/release).
+
+## New features
+
+* Add support for Valkey
+
+  * Add instrumentation for [Valkey](https://pypi.org/project/valkey/). Thanks [@jairhenrique](https://github.com/jairhenrique) for the contribution!
+  
+* Add TOML configuration file support  
+
+  * Support to read TOML configuration is now available. Thanks [@Tatsh](https://github.com/Tatsh) for the contribution!
+  
+* Add infinite tracing pb2 file for protobuf v5
+
+  * This updates the pb2 files used for infinite tracing to v5. 
+
+* Add support for attaching custom attributes to logs
+
+  * A new configuration option has been added called `application_logging.forwarding.custom_attributes` that will add custom attributes to all log events when set .
+
+* Add support for attaching labels to log events
+
+  * Labels as attributes can now be added to log events.  When `application_logging.forwarding.labels.enabled` is set the values in `labels` will be added to all log events. Labels can be excluded by setting `application_logging.forwarding.labels.exclude`.
+
+## Enhancements
+
+* Minor cleanup of HTTPX instrumentation
+
+  * Some minor cleanup was done of the HTTPX instrumentation.
+
+* Validate HTTP/2 support
+
+  * Added tests to ensure HTTP/2 support for the following frameworks:
+    * [Daphne](https://pypi.org/project/daphne/)
+    * [Hypercorn](https://pypi.org/project/Hypercorn/)
+    * [HTTPX](https://pypi.org/project/httpx/)
+    
+## Bug fixes
+
+* Collect module information from application periodically instead of at startup
+
+  * Previously, the New Relic Python agent collected the application's module information all at once during startup.  In some extreme cases this would cause a large spike in CPU usage. This enhancement allows the agent to do this incrementally throughout the application's operation, preventing potential memory spikes during agent startup.
```

Comment:
```suggestion
  * Previously, the New Relic Python agent collected the application's module information all at once during startup. In some extreme cases this would cause a large spike in CPU usage. This enhancement allows the agent to do this incrementally throughout the application's operation, preventing potential memory spikes during agent startup.
```